lib/core/test/unit/Cardano/Wallet/DB/Sqlite/TypesSpec.hs

```diff
@@ -136,5 +146,24 @@ instance Arbitrary TokenQuantity where
     arbitrary = genTokenQuantityFullRange
     shrink = shrinkTokenQuantityFullRange
 
+newtype Small a = Small { unSmall :: a } deriving (Eq, Show)
+newtype Large a = Large { unLarge :: a } deriving (Eq, Show)
+
+instance Arbitrary (Small TxMetadata) where
+    arbitrary = Small <$> genSmallTxMetadata
```

Comment:
I looked at the definitions of `genSmallTxMetadata` and `genTxMetadata` to try and see why they couldn't be replaced with `Test.QuickCheck.scale`.

The distinction seems to be that "small" generates non-recursive metadata values from the 3 primitive metadata types, and non-small generates metadata values of any depth.

So perhaps we could fix the naming. So remove `Large` and rename Small to something like `Primitive` or `Simple`.

Reply:
Fixed (hopefully) in dc709002de61c1f67f5074288341f33312d1e035.